### PR TITLE
[ft-158422941] add a user profile page

### PR DESCRIPTION
**Pull request by Ogollah**
#### What does this PR do?
Create a profile page
Create a create ride form

#### Description of Task to be completed
A user should be able to see their activities
A user should be able to add a new ride
A user should be able to see to see their details

#### How should I test it manually?
Clone this repository
Open your terminal
cd Ride-my-way
git checkout ft-profile-158422941
Open ride.html using your favorite browser

#### What PT stories?
[ft-158422941](https://www.pivotaltracker.com/story/show/158422941)
#### Screenshot

userprofile
pro

### CODE REVIEW

**Comment by meshack-mbuvi** [comment]
the profile looks good to me. Check on the best practice to naming of **PR title**

**Comment by kwameasiago** [comment]
The colour combination is amazing

**Comment by artorious** [comment]
Looking Good Bruv. 

Is this how you want to merge this PR though? Not `develop` branch?

`Ogollah wants to merge 1 commit into ft-ride-page-#158454339 from ft-profile-158422941`

**Comment by Xerrex** [comment]
Nice :zipper_mouth_face: out of words.

**Comment by LWanjiru** [comment]
Great improvement on UI @Ogollah Kudos! 

Curious question: What input does the `ride` field in your `create ride` form take?

Please consider following the `Best Practices` when creating a PR. This PR seems to have two separate features. All your stories should be atomic on the pivotal tracker as we discussed on Tuesday, and similarly, a PR should be for a single story. Please work on this moving forward.

File: UI/templates/profile.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    </div>
+
+    <!--footer fixed-->
+    <div class="footer">
+        <p>Ride-my-way &copy; 2018</p>
+    </div>
+<script language="JavaScript" src="../static/js/main.js" type="text/javascript"></script>
+</body>
+
+</html>
```
Comment (LWanjiru) [inline UI/templates/profile.html:131]: add new line here and at the end of all your files

**Review by LWanjiru — APPROVED** [review]

**Pull request merged**